gcs/CHANGES.md

```diff
@@ -82,7 +78,45 @@
 1.  The Google Cloud Storage Connector now can be used as a
     [Hadoop Credential Provider](https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/CredentialProviderAPI.html).
 
-1.  Added dependency on new storage client library: [google-cloud-storage](https://github.com/googleapis/java-storage/tree/main/google-cloud-storage).
+1.  Added dependency on the Cloud Storage Client Library
+    ([google-cloud-storage](https://github.com/googleapis/java-storage/tree/main/google-cloud-storage)).
+
+1.  Rename `fs.gs.rewrite.max.bytes.per.call` property to
+    `fs.gs.rewrite.max.chunk.per.call`.
+
+1.  Remove support of the deprecated `fs.gs.io.buffersize.write` property.
+
+1.  Add support for size suffixes (`k`, `m`, `g`, etc) in values of size-related
+    properties:
+
+    ```
+    fs.gs.inputstream.inplace.seek.limit
+    fs.gs.inputstream.min.range.request.size
+    fs.gs.outputstream.buffer.size
+    fs.gs.outputstream.pipe.buffer.size
+    fs.gs.outputstream.upload.cache.size
+    fs.gs.outputstream.upload.chunk.size
+    fs.gs.rewrite.max.chunk.per.call
+    ```
+
+1.  Remove `.ms` suffix from names and add support for time suffixes (`ms`, `s`,
+    `m`, etc) in values of time-related properties:
+
+    ```
+    fs.gs.http.connect-timeout
+    fs.gs.http.read-timeout
+    fs.gs.max.wait.for.empty.object.creation
+    fs.gs.outputstream.sync.min.interval
+    fs.gs.performance.cache.max.entry.age
+    ```
+
+1.  Change default values of properties:
```

Comment:
why are we changing these?

Reply:
Our perf team did analysis and 5s timeout values are mote performant. High memory consumption during write is a long-standing issue too - reducing buffer to 24m to mitigate it, in cost of minor perf impact for large file writes.